.github/workflows/one-click-hosting-release.yml

```diff
@@ -0,0 +1,140 @@
+name: One Click Hosting Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      pre_release:
+        type: boolean
+        description: 'Pre-release?'
+        default: true
+        required: false
+    branches:
+      - main
+env:
+  CHANNEL: 'cloud'
+  CHANGELOG_FILE: 'temp-changelog.txt'
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: checkout branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+      - name: Get Configuration
+        id: configuration
+        uses: rgarcia-phi/json-to-variables@main
+        with:
+          filename: './.github/workflows/config.json'
+          prefix: config
+      - name: Test permissions
+        uses: ./.github/workflows/permissions
+        with:
+          ENVIRONMENT: ${{ env.config_deployment_environment }}
+          DEPLOYMENT_PERMITTED: ${{ env.config_deployment_permitted }}
+          DEPLOYMENT_REPOSITORY_OWNER: ${{ env.config_deployment_repository_owner }}
+      - name: Install Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - name: Is pre-release?
+        id: is_pre_release
+        run: |
+          if [ ${{ github.event.inputs.pre_release }} == true ]; then
+            POSTFIX="-test"
+            echo "This is a prerelease execution"
+          else
+            POSTFIX=""
+            echo "This is a release execution"
+          fi
+          echo "POSTFIX=${POSTFIX}" >> $GITHUB_ENV
+      - name: Env Vars Configure
+        uses: ./.github/workflows/env-vars-configure
+        with:
+          CHANNEL: ${{ env.CHANNEL }}
+          REF: ${{ github.ref }}
+          POSTFIX: ${{ env.POSTFIX }}
+      - name: Bump Channel Version
+        uses: ./.github/workflows/bump-channel-version
+        with:
+          CLEAN_PACKAGE_VERSION: ${{ env.CLEAN_PACKAGE_VERSION }}
+          CHANNEL: ${{ env.CHANNEL }}
+          OVERRIDE_PACKAGE_VERSION: true
+          POSTFIX: ${{ env.POSTFIX }}
+      - name: Install Dependencies
+        run: npm ci
+      - name: Build plugin
+        uses: ./.github/workflows/build-plugin
+        with:
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+          BUILD_SCRIPT_PATH: "./.github/scripts/build-zip.sh"
+      - name: Generate changelog
```

Comment:
Please verify that the generated change log is correct & with correct versions, content & dates (if you didn't already)

Reply:
This is a cloud version, it generates a readme.txt file and the change log is visible in the release screen.
It doesn't update the regular changelog.

Reply:
Right, but if we generate a CL, lets make sure its correct (otherwise, lets not do that at all)